src/SDKs/StorSimple8000Series/StorSimple8000Series.Tests/StorSimple8000Series.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="10.1.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.8" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.6.0" />
+    <PackageReference Include="xunit" Version="2.2.0" />
```

Comment:
@shahabhijeet Hi, I replaced the individual package reference by the line to import "AzSdk.test.reference.props". Also, I added some other fields common across other SDKs test projects .csproj.  Still getting the same build failure. Any suggestions?

Also, for Resource Manager, I see the current version is 1.1.0-perview in above imported file. (similar version in other SDKs also) 
<PackageReference Include="Microsoft.Azure.ResourceManager" Version="[1.1.0-preview]" />
Do I still need to change it?